katcp/test/test_resource_client.py

```diff
@@ -16,7 +24,7 @@
 import tornado
 import mock
 
-from thread import get_ident as get_thread_ident
+from _thread import get_ident as get_thread_ident
```

Comment:
This looks a bit odd. Surely `_thread` is an implementation-detail module?

Reply:
@tockards see this comment regarding `_thread` import.

Reply:
@mmphego thank you.